custom_reports/columns.rst

```diff
@@ -381,7 +381,7 @@ dimensions to metrics or different dimensions.
     +-------------------+-----------------+----------------+-----------+
     |Max                |max              |int, float      |(as source)|
     +-------------------+-----------------+----------------+-----------+
-    |Average            |average          |int, float, bool|(as source)|
+    |Average            |average          |int, float, bool|float      |
```

Comment:
It looks like correction that should also be propagated to previous version docs. Can you confirm?

Reply:
If you feel like it, by all means :joy: